### ISSUE TITLE
07_geo.sql produces one NULL cas_id

### ISSUE THREAD

**Issue by pedrogit** [edwardsmarc/CASFRI#314]
We should get this NULL row in cas_all as well.

That means one source attribute needed to build cas_id is NULL. It's hard to find which one because what we normally use to get back to the source row (the cas_id) is NULL. The better way to find it I know is to rerun each inventory into their own table and find out which inventory has a NULL attribute (I would tend to try the last translated inventories first (ON, YT, SK)). Then you can search this inventory to find which row has a NULL:

```
SELECT * FROM rawfri.thisinventory
WHERE cas_id_att1 IS NULL OR cas_id_att2 IS NULL OR cas_id_att3 IS NULL OR cas_id_att4 IS NULL;
```

Another way would be to search the logs. The NULL value might have been trapped by a notNull() validation rule.

**Comment by pedrogit** [comment]
I found it by searching in the logs:

SELECT * FROM TT_ShowLastLog('translation', 'on_fim02_geo')
WHERE logentrytype != 'PROGRESS';

"INVALID_VALUE" "2948967" "Rule 'notNull(polyid=NULL)' failed for attribute 'CAS_ID' and reported error code 'NULL_VALUE'..."	2949415	1

**Comment by pedrogit** [comment]
I guess this is a duplicate of #304. I should have reload on02 before running the workflow. :-)

**Comment by edwardsmarc** [comment]
@pedrogit A nice test that the log is useful!

**Comment by pedrogit** [comment]
Yeah. I did not to wait an almost complete worflow to find the culpit.

**Comment by pedrogit** [comment]
Reloaded ON02 and found this issue to be a duplicate previously fixed by #304.
